external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/OffsetManager.java

```diff
@@ -49,10 +51,14 @@ public OffsetManager(TopicPartition tp, long initialFetchOffset) {
         LOG.debug("Instantiated {}", this);
     }
 
-    public void add(KafkaSpoutMessageId msgId) {          // O(Log N)
+    public void ack(KafkaSpoutMessageId msgId) {          // O(Log N)
```

Comment:
Isn't add more ambiguous now that there are two different methods for adding offsets (emit())?